src/OpenSpaceToolkit/Astrodynamics/Trajectory/Propagator.cpp

```diff
@@ -27,13 +29,18 @@ using ostk::astro::flight::system::dynamics::PositionDerivative;
 using ostk::astro::flight::system::dynamics::CentralBodyGravity;
 using ostk::astro::flight::system::dynamics::ThirdBodyGravity;
 using ostk::astro::flight::system::dynamics::AtmosphericDrag;
+using ostk::astro::trajectory::state::CoordinatesSubset;
 
-static const Shared<const Frame> gcrfSPtr = Frame::GCRF();
+static const Shared<const Frame> INTEGRATION_REFERENCE_FRAME = Frame::GCRF();
```

Comment:
Is this `ALL_CAPS` ok for static constants?

Reply:
Good call. We should change this.